apps/api-frontend/src/app/pages/search-pages/search.tsx

```diff
@@ -19,12 +20,12 @@ export default function SearchPage() {
       </header>
       {/* TODO add material design */}
       <div className="search-container">
-        <SearchBar onSearch={(query) => onSearch(query, db, history)} />
+        <SearchBar onSearch={(query) => onSearch(query, ScholarsDB.RUN_ALL, history)} />
         <div className="search-tips">
           <h4>A quick tip for getting the best search results</h4>
           <p>
-            Keep your search specific. So rather than searching for "fat", you
-            could "water and fat magnetic resonance" or "fat soluble vitamin
+            Keep your search specific. So, rather than searching for "fat", you
+            could search for "water and fat magnetic resonance" or "fat soluble vitamin
```

Comment:
are you expecting users to already know they are looking for magnetic resonance or soluble vitamins? these seem like big asks haha :)

Reply:
Nor do I :). I just searched fat and rote the first thing. I will change it